lib/damn.js

```diff
@@ -5,26 +5,26 @@ class Damn {
     this.client = new HttpClient(accessToken)
   }
 
-  getDailyDeviations () {
+  getDailyDeviations (qs) {
     return this.client
-      .get('browse/dailydeviations')
+      .get('browse/dailydeviations', Object.assign({}, qs))
```

Comment:
Why the shallow copy? Couldn't we simply pass on `qs`?
```js
.get('browse/dailydeviations', qs)
```

Reply:
We could, but it's generally best practice to clone objects since they are passed by reference. This is defensive programming that helps avoid unintended consequences if eg. perhaps `.get` modified the object somehow. Either way it's not a big deal.